source/index.js

```diff
@@ -128,7 +131,7 @@ module.exports = async (input = 'patch', options) => {
 			},
 			{
 				title: 'Installing dependencies using npm',
-				enabled: () => options.yarn === false,
```

Comment:
Please avoid doing any changes unrelated to the `--preview` flag in this PR. While the changes might be valuable and helpful to the project, they make reviewing this PR harder and we try to keep each PR's scope as narrow as possible.